### ISSUE TITLE
Keyboard navigation impossible because of missing focus styles

### ISSUE THREAD

**Issue by Diesmo** [duplicati/ngclient#349]
Trying to navigate the UI with a keyboard and tabbing is nearly impossible, as there are no visual indicator for most UI elements.

Further there are tab indexes on elements that are unnecessary. For example this tile and button, both have a tab index in this screenshot. 

<img width="1717" height="212" alt="Image" src="https://github.com/user-attachments/assets/2782d427-5956-4a61-84d1-ef1cf0780380" />

The tiles are even one of the few UI-elements that have a visual focus style. But after the next tab you are one the small "+ Add"-button without knowing it as there is no visual indication for this. And if you tab further you go the the "Details" and "List" visual toggles for the backup list. The interaction also works when pressing enter, but again there is no visual indicator what you have selected.

The same is also true for creating a backup.
The whole UI across all 5 steps when creating a new backup is not really navigate-able with keyboards only. There are also tab indexes missing or plain wrong. Simply try to create a new backup with they keyboard only and you will find all those spots in minutes.

At first I thought this might be a dark mode only problem as there might be no style or a black style for focused elements. But light mode is not much better, even though there are two or three elements that actually have a black focus style (in light mode). But the overall experience is still terrible in light mode.

The old UI is also not perfect and also has some tab index problems, but at least you always know what element is currently selected, even if it is the wrong one.
